src/pages/detail.js

```diff
@@ -0,0 +1,46 @@
+import { API_KEY, getWeatherInfo, THREE_HOURS_WEATHER_BASE_URL } from '../api/weather';
+import WeatherBoxView from '../views/weather-box';
+
+class Detail {
+  constructor(container) {
+    this.container = container;
+    this.weatherUrl = this.getWeatherUrl('Seoul');
+    this.weatherInfo = null;
+    this.render();
+    this.init();
```

Comment:
요런 이름으로 가져가도 괜찮을 것 같긴 하네요~
```suggestion
    this.mounted();
```